### ISSUE TITLE
Consider enabling Interactive Highlights in sims with Alternative Input

### ISSUE THREAD

**Issue by arouinfar** [phetsims/a11y-research#178]
In https://github.com/phetsims/greenhouse-effect/issues/205 we enabled Interactive Highlights in Greenhouse Effect due to user feedback in recent accessibility testing (report [here](https://drive.google.com/file/d/1ulEr8HcKOPcwTjMg5boF2_V3hbM5WDUc/view)). Users who use magnification reported insufficient hover behavior, and Interactive Highlights could be helpful in this situation. We will be doing another round of compatibility/UX testing later this month with Interactive Highlights enabled.

Should we consider enabling Interactive Highlights in all sims with Alternative Input support? There seems to be minimal overhead to turn on Interactive Highlights in sims that support Alternative Input, and it could potentially expand access to more users. Alternative Input is already becoming a standard feature in PhET sims, and with the recent changes to support localization, the Preferences dialog is also becoming standard.

**Comment by terracoda** [comment]
Great news that you found this in your interviews, too! It was two low vision users in interviews with me who commented that they really liked the keyboard highlights, so we decided to invent interactive highlights option for mouse users.

I think this would be an awesome addition to our baseline inclusion strategy!

I am not sure if Alt Input is required in order to enable the interactive highlights option. I think it is, but interactive highlights are different behaviourly than focus highlights, so not 100 percent sure. That would be a question for @jessegreenberg .

**Comment by terracoda** [comment]
The interactive highlight option is beneficial for many users who just like seeing what’s interactive more easily.

**Comment by emily-phet** [comment]
i agree with @terracoda. Interactive highlights are a relatively low implementation effort feature, and can be beneficial for many users across many different contexts. It would be great to have that as a standard feature for all new sims.

**Comment by arouinfar** [comment]
Thanks for your input @terracoda @emily-phet. I think we will move forward with making Interactive Highlights a standard feature. @jessegreenberg created the `interactiveHighlightsInitiallyEnabled` query parameter since other inclusive features have similar query parameters to enable them on startup.

> I am not sure if Alt Input is required in order to enable the interactive highlights option. I think it is, but interactive highlights are different behaviourly than focus highlights, so not 100 percent sure. That would be a question for @jessegreenberg .

I'm not sure about this, either. @jessegreenberg can you answer this one? Also, not sure what the next steps are. PSA at dev meeting?

**Comment by jessegreenberg** [comment]
> I am not sure if Alt Input is required in order to enable the interactive highlights option

It is not required, they can be done separately. When alt input is added a focus highlight designed for all interactive components in the sim. So the lift for adding Interactive Highlights to sims that support alt input is a bit less. That seems like a good starting point but if you would like other sims to have it, let me know.

>  Also, not sure what the next steps are. PSA at dev meeting?

Yep! PSA was made a couple of weeks ago and next steps are taking place here: https://github.com/phetsims/joist/issues/858.

Back to @arouinfar to review comments or to close if this is reasonable.

**Comment by arouinfar** [comment]
Thanks @jessegreenberg! I think we can go ahead and close this issue.
